src/Redux/Reducers/deleteHouseSlice.js

```diff
@@ -1,34 +1,36 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
 import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-extraneous-dependencies
 import axios from 'axios';
 
-export const createHouse = createAsyncThunk('houses/create', async (formData) => {
+export const deleteHouse = createAsyncThunk('houses/create', async (formData) => {
   const response = await axios.post('http://localhost:4000/api/v1/houses', { house: formData });
   return response.data;
 });
```

Comment:
If it is a delete house action, it must use the delete method for the Axios. But you are using the post method. It will never work unless you configure it according to the API endpoint.

Reply:
I don't know what the endpoint are yet....can you help with that please?